functions/hierarchyCheck.js

```diff
@@ -0,0 +1,44 @@
+/**
+Check if user is higher in guild's role hierarchy
+...Basically an easy way to stop mods from performing actions on admins, etc
+
+    Usage:
+        - client scope
+        - executor is the user object of the user executing the moderation command
+        - target is the user object of the user the moderation command is affecting
+        - guild is the guild object of the guild the command is being executed in
+    Returns:
+        - boolean true if executor is higher
+        - boolean false if executor is lower or equal
+        - boolean false if either user isn't in the guild
+
+        **/
+
+const func = async (client, executor, target, guild = null) => {
+  try {
```

Comment:
You also might want to fetch the users before the member, you'll this error if you try to fetch a member when the user itself is not cached.

```
User is not cached. Use Client.fetchUser first.'
```